documentation/service-rules/ecf1/lead-provider-api.md

```diff
@@ -2,4 +2,247 @@
 title: Lead provider API
 ---
 
-Lorem ipsum.
+Learn about how our payment engine works once it receives declarations from lead providers using the Early Career Framework API.
+
+These are the steps
+
+1. Provider submits a declaration.
+2. The system validates data.
+3. If invalid, errors are flagged up with corrective action suggestions.
+4. If valid, the declaration is created and linked to an output fee calculation.
+5. Service fees and banding are determined and applied.
+6. Final payments are calculated and stored.
```

Comment:
Do we store these? 🤔 I thought all the banding/payments stuff was determined at the time the statement is viewed?